File: app/src/main/java/hellojdh/wintercoding/IntroActivity.java
package hellojdh.wintercoding;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.support.annotation.Nullable;

import hellojdh.wintercoding.tool.BaseActivity;

/*
 * 인트로 화면
 */
public class IntroActivity extends BaseActivity {
    private static final String TAG = IntroActivity.class.getSimpleName();
    private static int INTRO_TIME = 1300;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_intro);

        goMain();
    }

    void goMain(){
        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent intent = new Intent(IntroActivity.this,MainActivity.class);
                startActivity(intent);
                finish();
            }
        },INTRO_TIME);
    }
}

File: app/src/main/java/hellojdh/wintercoding/IntroActivity.kt
package hellojdh.wintercoding

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import hellojdh.wintercoding.tool.BaseActivity

class IntroActivity : BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_intro)

        goMain()
    }

    private fun goMain(){
        val intent = Intent(this,MainActivity::class.java)
        Handler().postDelayed({
            startActivity(intent)
            finish()
        },INTRO_TIME)
    }

    companion object {
        const val INTRO_TIME : Long = 1300
    }
}